src/lib.rs

```diff
@@ -23,10 +23,13 @@ create_exception!(
     "Internal or filesystem error."
 );
 
+const EMPTY_STRING: String = String::new();
+
 // these need to match `watchfiles/main.py::Change`
 const CHANGE_ADDED: u8 = 1;
 const CHANGE_MODIFIED: u8 = 2;
 const CHANGE_DELETED: u8 = 3;
+const CHANGE_MOVED: u8 = 4;
```

Comment:
we should probably use an enum for these.

Either an enum with values, or:

```rs
enum Change {
  Added(String),
  Modified(String),
  Deleted(String),
  Moved(String, String),
}
```
Then implement `ToPyObject` on Change.

Reply:
Thanks, I will do that later. This is the first time I write Rust, so it's all new for me :smile:

Reply:
I see, then I'm impressed.

Let me know if you have more questions, or if you want some help with this.

Lots of examples of `ToPyObject` in pydantic-core.

Also github copolit is very useful when you're new to rust (or indeed not new to it), it makes some very helpful suggestions.